lib/core/src/connection_pool.cpp

```diff
@@ -237,11 +281,39 @@ namespace irods
         }
 
         try {
-            // NOLINTNEXTLINE(bugprone-unused-raii)
-            query<RcComm>{ctx.conn.get(), "select ZONE_NAME where ZONE_TYPE = 'local'"};
+            // Check if the number of requests executed via this connection has been reached.
+            if (options_.refresh_connections_after_n_retrievals) {
+                if (ctx.retrieval_count >= *options_.refresh_connections_after_n_retrievals) {
```

Comment:
Any reason these are separated?

Reply:
Just to avoid long lines and adding `// clang-format off/on`.

Reply:
If we don't like the way that clang-format formats these types of situations, do we need to revisit the clang-format configuration?

Reply:
I don't think we need to touch it.

The variable names are really long and this is just one way to satisfy the formatter.